test/SpokeUtils.ts

```diff
@@ -1,8 +1,47 @@
 import { ZERO_BYTES } from "../src/constants";
-import { getMessageHash, keccak256 } from "../src/utils";
+import { getMessageHash, getRelayEventKey, keccak256, randomAddress, toBN } from "../src/utils";
 import { expect } from "./utils";
 
+const random = () => Math.round(Math.random() * 1e8);
+
 describe("SpokeUtils", function () {
+  it("getRelayEventKey correctly concatenates an event key", function () {
+    const data = {
+      originChainId: random(),
+      destinationChainId: random(),
+      depositor: randomAddress(),
+      recipient: randomAddress(),
+      inputToken: randomAddress(),
+      inputAmount: toBN(random()),
+      outputToken: randomAddress(),
+      outputAmount: toBN(random()),
+      message: "0x",
```

Comment:
Nit can use randomBytes32 here to add entropy

Reply:
https://github.com/across-protocol/sdk/pull/860/commits/fd27acbcf7b39863277892558a17fe1c7d2ccdd9